iothub_client/src/uamqp_messaging.c

```diff
@@ -2,15 +2,19 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 #include <stdlib.h>
```

Comment:
Missing stdint.h for fixed size types
And probably inttypes.h since I can see you want to print fixed size types for errors.

Reply:
Done